src/main/java/frc/robot/sensors/Pigeon.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.sensors;
+import com.ctre.phoenix.sensors.*;
+import frc.robot.subsystems.DriveBase;
+
+public class Pigeon {
+
+    static PigeonIMU _pidgey;      // Pigeon IMU used to enforce straight drive
+
+    static {
+        //get talon
+        _pidgey = new PigeonIMU(DriveBase.getInstance().leftFrontTalon);
+
+        //setup gyro angle finder
+        _pidgey.setFusedHeading(0.0, 30);
```

Comment:
The side effect of running this command is that it will reset the pigeon heading.

That's OK if this is the convention we set, but everyone needs to be aware of it, e.g. the other students doing autonomous and vision commands.

The other way to do it is let the Pigeon keep track of its own heading and everyone uses their own variables to offset to what we want.